stryker/guides/stryker-diff-runner.md

```diff
@@ -0,0 +1,36 @@
+# stryker-diff-runner
+`stryker-diff-runner` is an utility script that allow you to run mutation testing on the part of the code that have diff with the `origin/master` branch.
+
+To get started using `stryker-diff-runner`, please follow these steps:
+1. Install required packages for the script to run: `npm i -D @stryker-mutator/core @stryker-mutator/api stryker-diff-runner`
+2. Install required packages for the targeted project typology (Angular, React.js, Vue.js or Node.js).
+3. Add at least the `mutate` section of the *stryker.conf.js* file with the files you want to have mutation on.
```

Comment:
With regards to step 3: would it be possible to use Stryker's logic of resolving the files to mutate? We could document it as a public api. If we would do that, you wouldn't need to search for the files yourself.

Reply:
I think Stryker resolves files according to `.gitignore` file when it is a Git project by default no ?
I use the `mutate` section of the Stryker config to inject files diff. I could do some improvements to do it even if that section is not present in the config file.
I also can inspect Stryker's file resolution strategy to be able to propose a more elegant solution.

Anyway, maybe it can be improve for next versions.